carina-utils/src/main/java/com/qaprosoft/carina/core/foundation/utils/messager/IMessager.java

```diff
@@ -0,0 +1,84 @@
+package com.qaprosoft.carina.core.foundation.utils.messager;
+
+import com.qaprosoft.carina.core.foundation.commons.SpecialKeywords;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.log4j.Logger;
+import org.testng.Reporter;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * ReportMessage is used for reporting informational and error messages both
+ * using needed loggers.
+ *
+ * @author brutskov
+ */
+
+public interface IMessager {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move constants to a class or enum. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://demo.qaprosoft.com/sonarqube/coding_rules#rule_key=squid%3AS1214)